src/tests/dfio/daos_fio.c

```diff
@@ -186,12 +186,11 @@ static void
 daos_fio_cleanup(struct thread_data *td)
 {
 	struct daos_data *dd = td->io_ops_data;
-	int rc;
 
-	rc = dfs_umount(dd->dfs);
-	rc = daos_cont_close(dd->coh, NULL);
-	rc = daos_pool_disconnect(dd->poh, NULL);
-	rc = daos_fini();
+	dfs_umount(dd->dfs);
```

Comment:
that's not the right way to fix the build error.
you should keep the rc and add a DCHECK after to check for failure.

Reply:
struct ioengine_ops {
…
        void (*cleanup)(struct thread_data *);
…
}
Adding DCHECK here has to change daso_fio_cleanup() return value from void to int, which requires to
change the struct ioengine_ops definition in fio code.

Reply:
We should at least check the return value and print an error, even if the error code is not propagated up the stack. Let's land this patch, but please push another patch to fix this. Silently discarding errors is never a good idea.